File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/GenresResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.ryanrvldo.moviecatalogue.data.source.model.Genre;

import java.util.ArrayList;

import lombok.Data;

@Data
public class GenresResponse {
    @SerializedName("genres")
    @Expose
    private ArrayList<Genre> genres;
}

File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/remote/response/GenresResponse.kt
package com.ryanrvldo.moviecatalogue.data.remote.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.ryanrvldo.moviecatalogue.data.model.Genre
import java.util.*

data class GenresResponse(
    @Expose @SerializedName("genres") val genres: ArrayList<Genre>
)